org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/MavenCentralIdentifier.java

```diff
@@ -85,30 +86,43 @@ private ArtifactKey identifySha1(String sha1, IProgressMonitor monitor) {
 	}
 
 	private ArtifactKey find(String searchUrl, IProgressMonitor monitor) throws IOException, InterruptedException {
```

Comment:
I think these thrown exceptions could be removed since we handle them in the body now.

Reply:
Fixed.